negotiator/negotiator.py

```diff
@@ -407,7 +407,7 @@ def negotiate(self, accept=None, accept_language=None, accept_encoding=None, acc
         - accept_charset - HTTP Header: Accept-Charset; not currently supported in negotiation
         - accept_packaging - HTTP Header: Accept-Packaging (from SWORD 2.0); a URI only, no q values
         
-        If verbose=True, then this will print to stdout
+        If verbose=True, then this will print(to stdout
```

Comment:
This looks like mistake in search & replace :)

Reply:
Oh yeah, I've fixed this and updated PR. Thank you for reviewing it.